WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -316,6 +336,15 @@ public void onCreate(Bundle savedInstanceState) {
         }
     }
 
+    @Override public void onActivityCreated(@Nullable Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+        // we need to pass activity, since this fragment extends Android Native fragment (we can pass `this` as soon as
+        // this fragment extends Support fragment.
+        mViewModel = ViewModelProviders.of((FragmentActivity) getActivity(), mViewModelFactory)
+                                       .get(ReaderPostListViewModel.class);
+        mViewModel.start(mCurrentTag);
```

Comment:
I got a crash on this line, when clicking the "post author" part of the card
[![Image from Gyazo](https://i.gyazo.com/68c7068efe2d93666c6a195241978040.gif)](https://gyazo.com/68c7068efe2d93666c6a195241978040)

Tested on Genymotion Pixel 2 emultor on android 8.0 (API 26)
```
08-22 02:11:34.090 8843-8843/org.wordpress.android E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.wordpress.android, PID: 8843
    java.lang.RuntimeException: Unable to start activity ComponentInfo{org.wordpress.android/org.wordpress.android.ui.reader.ReaderPostListActivity}: java.lang.IllegalArgumentException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkParameterIsNotNull, parameter tag
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2817)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2892)
        at android.app.ActivityThread.-wrap11(Unknown Source:0)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1593)
        at android.os.Handler.dispatchMessage(Handler.java:105)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6541)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767)
     Caused by: java.lang.IllegalArgumentException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkParameterIsNotNull, parameter tag
        at org.wordpress.android.ui.reader.viewmodels.ReaderPostListViewModel.start(Unknown Source:2)
        at org.wordpress.android.ui.reader.ReaderPostListFragment.onActivityCreated(ReaderPostListFragment.java:345)
        at android.app.Fragment.performActivityCreated(Fragment.java:2620)
        at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1296)
        at android.app.FragmentManagerImpl.addAddedFragments(FragmentManager.java:2415)
        at android.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2194)
        at android.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManager.java:2148)
        at android.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2049)
        at android.app.FragmentManagerImpl.dispatchMoveToState(FragmentManager.java:3044)
        at android.app.FragmentManagerImpl.dispatchActivityCreated(FragmentManager.java:2991)
        at android.app.FragmentController.dispatchActivityCreated(FragmentController.java:178)
        at android.app.Activity.performCreateCommon(Activity.java:6969)
        at android.app.Activity.performCreate(Activity.java:6977)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1213)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2770)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2892) 
        at android.app.ActivityThread.-wrap11(Unknown Source:0) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1593) 
        at android.os.Handler.dispatchMessage(Handler.java:105) 
        at android.os.Looper.loop(Looper.java:164) 
        at android.app.ActivityThread.main(ActivityThread.java:6541) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767) 
```

Reply:
I can't believe, I missed this.... Thanks!! :)